src/index.ts

```diff
@@ -68,7 +68,7 @@ function keydown(fn: (event: KeyboardEvent) => void): (event: KeyboardEvent) =>
   }
 }
 
-type Style = {
+export type Style = {
```

Comment:
Don't think this is a problem but is there a reason this got exported?

Reply:
Just leaving the door open for future type-checking possibilities. This is the type that a subclass needs to produce.